sdks/java/io/jdbc/src/test/java/org/apache/beam/sdk/io/jdbc/JdbcIOTest.java

```diff
@@ -253,6 +252,43 @@ public void setParameters(PreparedStatement preparedStatement)
      pipeline.run();
    }
 
+   @Test
+   @Category(NeedsRunner.class)
```

Comment:
AFAIR `NeedsRunner` is not required there.

Reply:
Done, thanks, removed it from other tests too!